src/io.rs

```
@@ -109,6 +118,22 @@ where
             Ok(())
         })?;
     }
+    // For poi_properties.txt, it's a bit different: If the file is not
+    // present, it does not mean it is an error.
+    if let Ok(zipper) = zip.by_name("poi_links.txt") {
+        read_csv(zipper).try_for_each::<_, Result<_>>(|rec| {
+            let poi_link: PoiLink = rec?;
+            let poi = pois.get_mut(&poi_link.poi_parent_id).ok_or_else(|| {
+                anyhow!(
+                    "in file '{}', cannot find poi '{}' for link insertion",
+                    path.as_ref().display(),
+                    &poi_link.poi_parent_id
+                )
+            })?;
```

I'm wondering if we should return an error when an identifier doesn't exist... or if we should just ignore it and continue?

-----

Je me demande si on ne devrait pas simplement ignorer dans le cas d'un identifiant inexistant, au lieu d'interrompre avec une erreur.

---

Done

---

We're finally logging incorrect POI identifier, but will continue to process the rest of the file, see [3fbbefb](https://github.com/hove-io/navitia-poi-model/pull/11/commits/3fbbefbfc6373f967a2f980bcef7b100632b871e) with tests about logging.